assets/stylesheets/main.css

```diff
@@ -20,12 +20,15 @@ h3 {
 }
 .read{
```

Comment:
Put a space before the opening brace { in rule declarations

https://github.com/airbnb/css#formatting